projects/components/src/link/link.component.ts

```diff
@@ -9,9 +9,8 @@ import { isNil } from 'lodash-es';
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
     <a
-      *ngIf="this.navigationPath"
       class="ht-link"
-      [ngClass]="{ disabled: this.disabled }"
+      [ngClass]="{ disabled: (this.disabled || this.navigationPath)}"
```

Comment:
Wouldn't we want to negate this? this would eval true if not disabled and a path is provided.

Reply:
oh my bad